src/wasm/wasm-type.cpp

```diff
@@ -949,7 +955,9 @@ FeatureSet Type::getFeatures() const {
             if (sig.results.isTuple()) {
               feats |= FeatureSet::Multivalue;
             }
-          } else if (heapType->isContinuation()) {
+          }
```

Comment:
Can something be both a signature and a continuation?

Reply:
No, but the original else-if up above was incorrect because it is possible for something to have a non-trivial rec group or have a declared supertype or be open and also be a signature. Rather than mix ifs and if-elses, I just made everything an if.

Reply:
Yeah, that part made sense to me. But maybe it's still good to have an if-else chain after it, for the disjoint cases? That seems clearer to me (even if it might lead to some overlapping code).

Reply:
Fair enough.